src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -578,25 +578,33 @@ ServerHandlerImp::processRequest (Port const& port,
     bool batch = false;
     if (jsonOrig.isMember(jss::method) && jsonOrig[jss::method] == "batch")
         batch = true;
-    auto size = batch ? jsonOrig[jss::params].size() : 1;
+    auto size = batch
+              ? jsonOrig.isMember(jss::params)
+                ? jsonOrig[jss::params].size() : 0
+              : 1;
```

Comment:
I'd rewrite this. Nested ternary expressions are too hard to read. I'd use an immediately evaluated lambda, but however you want to simplify this is fine.